vault/notes/new-home-page.md

```diff
@@ -0,0 +1,112 @@
+---
+title: "New Home Page"
+created: 2023-03-28
+---
+
+## Ever asked yourself ... How can I live a happier life? Create a better society?
+
+We’re an open community dedicated to the art of wiser living and social transformation. We favor approaches that prioritize inner development and cultural change in a rigorous, practical way.  
+
+Sign up to our newsletter for latest updates, projects 
+and community action.
+
+[EMAIL SIGN UP ]
+
+![[../assets/images/lifeitself-landingpage.webp]]
+
+In the past four decades, a confirmation and expansion of wisdom traditions
+by scientific rigor has converged with an ever deepening polycrisis.
+
+There has never been a more important time to “grow up”, both personally and collectively. A
+ paradigm shift is underway and we all have a part to play. 
+
+The big question is…. how? 
+
+Life itself is a home for people who are dedicated to rigorous enquiry a
+nd deliberate action for inner growth and cultural evolution.
+
+Inspired by integral theory, zen buddhism and other key concepts, we cultivate these ideas and 
+apply them to life itself, through deliberately developmental programs and community building.
+
+### [[../about |Learn More]]
+
+____________
+
+## Join The Community
+
+The Life Itself online community space is a fantastic place to get started. 
+
+We have regular community calls, presentation, a jobs board and events board.
+
+## [Join Here](https://chat.whatsapp.com/JNJCTZugNQn1fq89xbHtfA) 
+
+![[../assets/images/Whatsapp General Chat QR Code.png]]
+
+## What We're Up To
+
+### Major Projects
+
+- Labs
+- Hubs
+- Ecosystem
+- Podcast
+
+### Key Articles
+
+- Add
+- Key 
+- Articles
+- Here
+
+### Active Projects
+
+- Conscious Parenting
+- Residencies
+- Podcast
+- Community 
+
+## Try Out Living In Community 
+
+- [[../setting-the-world-to-rights-residency]]
+- [[../ouroboros-residency]]
+- [[../sympoiesis/conscious-food-retreats-culture-broths-1]]
+
+### [[../upcoming-residencies-gatherings |See More Opportunities]]
+
+## Dive Into A Topic
+
+We break down big ideas into accessible guides so you can learn and implement them faster.
+
+### Integral Theory
+### Ecosystem Mapping
+### Web3 
+
+## Latest Updates
+
+Slider including blog posts and podcasts
+
+Sign Up to our newsletter and don't miss out on any new updates
+[[sign up form]]
+
+![[../assets/images/profile-image-zaib.jpg]]
+
+## Meet The Team
+
+Our team consists of artists, researchers,
+technologists and community builders. Despite
+these varied backgrounds, each shares a 
+commitment to being wise and taking practical
+action to solve the collective challenges facing
+our society and culture. Read more about our 
+people here.
+
+### [[../people |Read More]]
+
+## Our Journey So Far
+
+Timeline of our journey including links to resources along the way.
```

Comment:
Can we have a go at actually drafting a basic version of this? I think we have material from 2020 gathering we could use and could do a quick interview.

Also is this about a **history** (timeline) or a **story** (as in our motivation ...)